tests/algorithms/lstm_classifier/lstm_classifier_test.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import sys
+import os
+from unittest.mock import MagicMock
+
+sys.path.append(os.path.realpath("."))
+
+import src.algorithms.lstm_classifier.lstm_classifier as lstm_classifier
+
+
+class LstmClassifierTest(unittest.TestCase):
+    def test_get_articles_and_labels(self):
+        review = [
+            {
+                "message": "I just heard about this budgeting app and I gave it a try and done.",
+                "derived-insight": {"category": "Application"},
+            }
+        ]
+        result = lstm_classifier.get_articles_and_labels(review)
+
+        # Check if stop words have been removed from the review message
+        self.assertFalse("and" in result[0][0])
```

Comment:
Can we also check that it hasn't removed the non stop-words ?

Reply:
Can we also check if NA strings have been removed?

Reply:
Are we removing or just ignoring the NA strings? [here](https://github.com/intuit/fawkes/blob/8f4a0183b20efebc973ad6157da599439ad9dcc2/src/algorithms/lstm_classifier/lstm_classifier.py#L69)

Reply:
@Konoha-14 article.strip() doesn't seem to be enough? When the test message is " " it fails to clean vs when the message is just "" (Note the space missing).